### PR TITLE
Add Kafka Cluster converter

### PR DESCRIPTION
**Pull request by sergenyalcin**
### Description of your changes

Updates dependencies by using the new submodules and adds the Kafka Cluster converter

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: converters/provider-aws/kafka/cluster.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		"spec.forProvider.encryptionInfo", "spec.forProvider.loggingInfo", "spec.forProvider.openMonitoring"); err != nil {
+		return nil, errors.Wrap(err, "failed to copy source into target")
+	}
+	target.Spec.ForProvider.Region = &source.Spec.ForProvider.Region
+	if source.Spec.ForProvider.CustomConfigurationInfo != nil {
+		target.Spec.ForProvider.ConfigurationInfo = make([]targetv1beta1.ConfigurationInfoParameters, 1)
+		target.Spec.ForProvider.ConfigurationInfo[0].Arn = source.Spec.ForProvider.CustomConfigurationInfo.ARN
+		target.Spec.ForProvider.ConfigurationInfo[0].Revision = common.PtrFloat64FromInt64(source.Spec.ForProvider.CustomConfigurationInfo.Revision)
+	}
+	if source.Spec.ForProvider.CustomBrokerNodeGroupInfo != nil {
```
Comment (erhancagirici) [inline converters/provider-aws/kafka/cluster.go:40]: nit:
the following fields are newly introduced at the target API. might be good idea to document it for the consumers.
`.Spec.ForProvider.BrokerNodeGroupInfo[0].AzDistribution`
`.Spec.ForProvider.BrokerNodeGroupInfo[0].EBSVolumeSize`
also it started to accept xp reference and selector
`.Spec.ForProvider.EncryptionInfo[0].EncryptionAtRestKMSKeyArnRef`
`.Spec.ForProvider.EncryptionInfo[0].EncryptionAtRestKMSKeyArnSelector`

**Review by erhancagirici — APPROVED** [review]
LGTM in general, left a few non-critical notes

File: converters/common/common.go [hunk 2]
```diff
@@ -37,8 +37,14 @@ func PtrFromString(s string) *string {
 	return &s
 }
 
-// PtrFloat64 returns the parameter of the type float64 in the pointer type.
-func PtrFloat64(i *int32) *float64 {
+// PtrFloat64FromInt32 returns the parameter of the type int32 in the pointer type.
+func PtrFloat64FromInt32(i *int32) *float64 {
+	a := float64(*i)
```
[the remark this answers is not in the document]
Comment (sergenyalcin) [reply converters/common/common.go:45]: Done!

File: converters/provider-aws/kafka/cluster.go (hunk 1 above)
Comment (sergenyalcin) [reply converters/provider-aws/kafka/cluster.go:40]: Added documentation!

**Pull request merged**